pkg/sql/sem/builtins/builtins.go

```diff
@@ -5296,8 +5293,9 @@ value if you rely on the HLC for accuracy.`,
 				tableID := int(tree.MustBeDInt(args[0]))
 				indexID := int(tree.MustBeDInt(args[1]))
 				g := tree.MustBeDGeography(args[2])
-				// TODO(ajwerner): This should be able to use the normal lookup mechanisms.
-				tableDesc, err := catalogkv.MustGetTableDescByID(ctx.Context, ctx.Txn, ctx.Codec, descpb.ID(tableID))
+				cf := descs.NewBareBonesCollectionFactory(ctx.Settings, ctx.Codec)
+				descsCol := cf.MakeCollection(descs.NewTemporarySchemaProvider(ctx.SessionDataStack))
```

Comment:
I suspect that the `TemporarySchemaProvider` is wholly unnecessary here but whatever.